samples/lib/main.dart

```diff
@@ -77,6 +77,11 @@ class _AppRouter extends RootStackRouter {
     AutoRoute(path: '/line-calendar/default', page: LineCalendarRoute.page),
     AutoRoute(path: '/modal-sheet/default', page: ModalSheetRoute.page),
     AutoRoute(path: '/modal-sheet/draggable', page: DraggableModalSheetRoute.page),
+    AutoRoute(path: '/pagination/default', page: PaginationRoute.page),
+    AutoRoute(path: '/pagination/siblings', page: PaginationSiblingsRoute.page),
+    AutoRoute(path: '/pagination/hide-first-last', page: PaginationHideFirstLastRoute.page),
```

Comment:
nitpick: I think show-edges is a better name (don't forget to rename the class too)